cg/apps/sequencing_metrics_parser/parsers/bcl2fastq.py

```diff
@@ -16,93 +19,175 @@
 LOG = logging.getLogger(__name__)
 
 
-def parse_bcl2fastq_sequencing_metrics(flow_cell_dir: Path) -> List[Bcl2FastqSampleLaneMetrics]:
-    """Parse metrics for a flow cell demultiplexed with Bcl2fastq."""
-    tile_sequencing_metrics: List[
-        Bcl2FastqSampleLaneTileMetrics
-    ] = parse_bcl2fastq_raw_tile_metrics(flow_cell_dir)
+def scale_paired_reads_to_total_reads(paired_reads: int) -> int:
```

Comment:
```suggestion
def scale_paired_reads_to_total_reads(paired_reads: int) -> int:
     """Multiply the read count of R1 by two to represent the total amount of reads for both read pairs R1 and R2"""
```

Reply:
Not sure if the caller should care about the details, but added this 👍